recipes/molpopgen-analysis/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version="0.8.8" %}
+{% set hash="f9ef9e0a90fce2c0f4fe462d6c05e22fef22df1c23b63a7c64ad7b538f6e8bb0" %}
+
+package:
+  name: molpopgen-analysis
+  version: {{ version }}
+
+source:
+  fn:  analysis-{{ version }}.tar.gz 
+  url: https://github.com/molpopgen/analysis/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc  
+    - gsl {{ CONDA_GSL }}*
+    - libsequence 1.8.4
+  run: 
+    - libgcc
+    - gsl {{ CONDA_GSL }}*
+    - libsequence 1.8.4
+
+test:
+  commands:
+    - command -v compute
+    - command -v gestimator
+    - command -v kimura80
+    - command -v polydNdS
+    - command -v sharedPoly
+    - command -v descPoly
+    - command -v HBKpermute
+    - command -v MKtest
+    - command -v rsq
+
+about:
+  home: http://http://molpopgen.github.io/libsequence/
```

Comment:
Is this link correct?

Reply:
The recipe fail if the link is incorrect, meaning the build was unsuccessfull

Reply:
But wouldn't this link be better? https://github.com/molpopgen/analysis

Reply:
Copy/paste error :p